app/helpers/browse_helper.rb

```diff
@@ -1,21 +1,10 @@
 module BrowseHelper
-  def display_popular_links_for_slug?(slug)
-    popular_links_data(slug).present?
-  end
-
-  def popular_links_data(slug)
-    if I18n.exists?("browse.popular_links.#{slug}", :en)
-      I18n.t("browse.popular_links.#{slug}")
-    end
-  end
-
-  def popular_links_for_slug(slug)
-    links = popular_links_data(slug)
-    count = links.length
-    links.map.with_index(1) do |link, index|
+  def with_tracking_data(popular_content)
```

Comment:
Could you update the commit message with why / when the tracking data was removed. Perhaps a link to what it was before? I'm assuming this has something to do with the switch from UA to GA4?

Reply:
Ah it wasn't removed, I just meant adding it back onto the views. I've reordered the commits so hopefully it makes more sense to you now?

Reply:
Ok, yes, this does make more sense now. Thanks!